sycl/include/CL/sycl/group.hpp

```diff
@@ -376,5 +376,19 @@ template <int Dimensions = 1> class group {
   }
 };
 
+namespace detail {
+template <int Dims> group<Dims> store_group(const group<Dims> *g, bool write) {
```

Comment:
It can, but the intention was to improve readability. Just having thing wrappers over `store` functions makes the action explicit for the specific class. Furthermore, we have such overloads with concrete type. That gives us static type checking without SFINAE or something. It's harder to make a mistake.

Reply:
Ok.